html-relative-links/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Home Page</title>
+    <style>
+      img {
+        width: 300px;
+      }
+    </style>
+  </head>
+  <body>
+    <ul>
+      <li><a href="#">Home Page</a></li>
+      <li><a href="./about-me/about-me.html">About Me</a></li>
+      <li><a href="./projects/projects.html">Projects</a></li>
+      <li><a href="./contact-us/contact-us.html">Contact Us</a></li>
+    </ul>
+    <p>Hello Homepage!</p>
+    <img
+      src="https://images.pexels.com/photos/1181703/pexels-photo-1181703.jpeg"
```

Comment:
This should be ponting to one of the images we provided for you in the `images` directory